prime-router/src/main/kotlin/history/SubmissionHistory.kt

```diff
@@ -336,9 +336,19 @@ class DetailedSubmissionHistory(
         }
 
         // Grab destinations from the "translate" action, if the submission made it that far
-        descendants.filter { it.actionName == TaskAction.translate }.forEach { descendant ->
+        descendants.filter { it.actionName == TaskAction.translate }.distinct().forEach { descendant ->
             descendant.destinations.forEach { dest ->
```

Comment:
It might be useful to distinguish between the Submission History destinations property and dest the destination within the list of descendant destinations. I was trying to come up with variable names and honestly the only one that seemed descriptive was descendantDest or descendantDestination. 

Feel free to leave it as is if others aren't confused by it